test-apps/display-test-app/src/frontend/App.ts

```diff
@@ -235,7 +235,7 @@ export class DisplayTestApp {
         ],
         /* eslint-disable @typescript-eslint/naming-convention */
         mapLayerOptions: {
-          MapBoxImagery: configuration.mapBoxKey ? { key: "access_token", value: configuration.mapBoxKey } : undefined,
+          MapboxImagery: configuration.mapBoxKey ? { key: "access_token", value: configuration.mapBoxKey } : undefined,
```

Comment:
I don't understand why this changed case. The property in `MapLayerOptions` is `MapBoxImagery`, not `MapboxImagery`.
I think you must be looking it up by MapBoxImageryMapLayerFormat.formatId, which is "MapboxImagery".
I will rename the MapLayerOptions property accordingly.

The compiler can't catch these mistakes due to the index signature on MapLayerOptions.

Reply:
Following my change the MapBox views in ImageTests produce map imagery.